src/services/enum-utilities.ts

```diff
@@ -96,6 +96,46 @@ export default class EnumUtilities {
     return (item.name === FilingTypes.AGM_LOCATION_CHANGE)
   }
 
+  /** Returns True if filing is an Amalgamation Application. */
```

Comment:
These were simply moved up from below so the methods are (mostly) in alphabetized.